pages/1.13/cli/command-reference/dcos-auth/dcos-auth-login/index.md

```diff
@@ -3,44 +3,32 @@ layout: layout.pug
 navigationTitle:  dcos auth login
 title: dcos auth login
 menuWeight: 2
-excerpt: Authenticating to DC/OS
-
+excerpt: Logging in to your DC/OS cluster
```

Comment:
I think we might better say log-in instead of logging in (I read it as actually logging from log files), our docs team may have a better articulation